packages/elements-core/src/components/Docs/Model/Model.spec.tsx

```diff
@@ -21,4 +22,16 @@ describe('Model', () => {
     expect(screen.getByRole('heading', { name: /example/i })).toBeInTheDocument();
     expect(container).toHaveTextContent('"propA": "valueA"');
   });
+
+  it('displays description at top of doc for objects', async () => {
+    const { container } = render(<Model data={exampleSchema} />);
+
+    expect(container).toHaveTextContent(/example schema description/i);
+  });
+
+  it('does not display description at top of doc for non-objects', async () => {
```

Comment:
You're not rendering anything here. First you need create another schema, that is not an object (e.g. string)
```json
{
"type": "string",
"description": "I am a string!"
}
```

Then render Model component using it, and then do the assertion - check that the `textbox` containing `I am a string!` isn't there

I don't think that we need to check if the description is present in schema's row though - it should be covered by JSV's test suite.

Reply:
AHA yes I was trying to go based off of the original first test but if I just render the screen it works how I want it to. Thank you for pointing this out!

Reply:
Addressed in 4158e71